doc/user/content/sql/system-catalog.md

```diff
@@ -101,6 +124,67 @@ Field  | Type       | Meaning
 `oid`  | [`oid`]    | A [PostgreSQL-compatible OID][oid] for the database.
 `name` | [`text`]   | The name of the database.
 
+### `mz_dataflow_channels`
+
+The `mz_dataflow_channels` source describes the communication channels between
+[dataflow] operators. A communication channel connects one of the the outputs of a
+source operator to one of the inputs of a target operator.
+
+Field         | Type       | Meaning
+--------------|------------|--------
+`id`          | [`bigint`] | The ID of the channel.
```

Comment:
We have elsewhere in our code (the foreign keys relation) information about what these columns should be linked to. Is there a good way to do that here (e.g. to call out that this column is a primary key, and can be linked over to `mz_dataflow_operator_addresses` which has it as a primary key)? If there is a format that we like, I'm happy to go through and decorate all of them that I know about.

Reply:
So I did this in the other direction, at least, for the foreign keys that correspond to these!